gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -128,12 +138,14 @@ def _resource_owner_tenant_id(self):
 
     def _get_resource_owner_context(self):
         if cfg.CONF.heat_driver.is_service_admin_owned:
-            tenant_id = self._resource_owner_tenant_id()
+            tenant_id = None
             user, pwd, tenant_name, auth_url =\
                 self.keystoneclient.get_keystone_creds()
             auth_token = self.keystoneclient.get_scoped_keystone_token(
                 user, pwd, tenant_name, tenant_id)
```

Comment:
I will revert this back..  in earlier experiments wanted to optimize this method, but this is not being used now by optimized path. so will revert back.